core/modules/wmgr/python/dbMgr.py

```diff
@@ -0,0 +1,692 @@
+    if chunkId < 0:
+        raise ExceptionResponse(400, "InvalidArgument", "Chunk ID argument is negative")
+
+    # check that table exists
+    dbConn = Config.instance().dbConn()
+    if not dbConn.tableExists(tblName, dbName):
+        raise ExceptionResponse(404, "TableMissing", "Table %s.%s does not exist" % (dbName, tblName))
+
+    chunkRepr = None
+
+    # chunk data table
+    # TODO: we need some central location for things like this
+    table = tblName + '_' + str(chunkId)
+    if dbConn.tableExists(table, dbName):
+
+        # drop table
+        _log.debug('chunkDelete(): drop chunk table: %s', table)
+        dbConn.dropTable(table, dbName)
+
+        chunkRepr = _chunkDict(dbName, tblName, chunkId)
+        chunkRepr['chunkTable'] = True
+
+    # overlap data table
+    # TODO: we need some central location for things like this
+    table = tblName + 'FullOverlap_' + str(chunkId)
+    if dbConn.tableExists(table, dbName):
+
+        # drop table
+        _log.debug('chunkDelete(): drop chunk table: %s', table)
+        dbConn.dropTable(table, dbName)
+
+        if chunkRepr is not None:
+            chunkRepr['overlapTable'] = True
+        else:
+            # chunk data table is missing
+            _log.error('chunkDelete(): Chunk does not exist, but overlap does')
+            raise ExceptionResponse(404, "ChunkDeleteFailed", "Cannot delete chunk data table",
+                                    "Chunk %s is not found for table %s.%s (but overlap table was deleted)" %
+                                    (chunkId, dbName, tblName))
+
+    if chunkRepr is None:
+        # nothing found
+        _log.error('chunkDelete(): Chunk does not exist')
+        raise ExceptionResponse(404, "ChunkDeleteFailed", "Cannot delete chunk data table",
+                                "Chunk %s is not found for table %s.%s" % (chunkId, dbName, tblName))
+
+    return json.jsonify(result=chunkRepr)
+
+
+@dbService.route('/<dbName>/tables/<tblName>/data', methods=['POST'])
+@dbService.route('/<dbName>/tables/<tblName>/chunks/<int:chunkId>/data', methods=['POST'])
+@dbService.route('/<dbName>/tables/<tblName>/chunks/<int:chunkId>/overlap', methods=['POST'])
+def dataLoad(dbName, tblName, chunkId=None):
+    """
+    Upload data into a table or chunk using the file format supported by mysql
+    command LOAD DATA [LOCAL] INFILE.
+
+    Not implemented yet.
+    """
+
+    _log.debug('request: %s', request)
+    _log.debug('tableDataLoad() POST => load data into chunk or overlap')
+
+    # validate params
+    _validateDbName(dbName)
+    _validateTableName(tblName)
+
+    _log.error('dataLoad(): method not implemented yet')
+    raise ExceptionResponse(501, "NotImplemented", "Data loading operations are not implemented yet")
+
+
+@dbService.route('/<dbName>/tables/<tblName>/index', methods=['GET'])
+@dbService.route('/<dbName>/tables/<tblName>/chunks/<int:chunkId>/index', methods=['GET'])
+def getIndex(dbName, tblName, chunkId=None):
```

Comment:
Interesting, this index will be quite big, do you plan to use this feature to create the full index?